tox.ini

```diff
@@ -34,12 +34,13 @@ description = Invoke nile-coverage to generate coverage report
 skip_install = True
 setenv =
     {[testenv]setenv}
+    PIP_EXTRA_INDEX_URL=https://test.pypi.org/simple/
     CAIRO_PATH = src
 deps =
     {[testenv]deps}
-    nile-coverage==0.2.2
+    nile-coverage==0.2.3b0
```

Comment:
```suggestion
    nile-coverage==0.2.3b0
```
why beta?

Reply:
I released a beta version in test.pypi to test the integration with cairo-contracts. This PR is not intended to be merged as it is (I will update after releasing 0.2.3).